NRasterizer.Tests/FakeGlyphRasterizer.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NRasterizer.Tests
+{
+    public class FakeGlyphRasterizer : IGlyphRasterizer
+    {
+        public int BeginReadCallCount { get; private set; }
+        public int CloseFigureCallCount { get; private set; }
+        public int ContourCount { get; private set; }
+        public int EndReadCallCount { get; private set; }
+        public int FlushCallCount { get; private set; }
+        public List<Point<double>> Points { get; set; } = new List<Point<double>>();
+        public int Resolution { get; set; } =  72;
+
+        public void BeginRead(int countourCount)
+        {
+            this.BeginReadCallCount++;
+            this.ContourCount = countourCount;
```

Comment:
no problem i'll drop them (going to be hard jumping back and fourth between ImageSharp and this project, both have very different coding styles so I apologies now for when I constantly use the wrong style 😄  )